dashboard/app.R

```diff
@@ -331,6 +333,10 @@ server <- function(input, output, session) {
     
     filteredScoreDf$ahead = factor(filteredScoreDf$ahead, levels = c(1, 2, 3, 4), 
                                     labels = c("Horizon: 1 Week", "Horizon: 2 Weeks", "Horizon: 3 Weeks", "Horizon: 4 Weeks"))
+    set.seed(colorSeed)
+    forecasterRand <- sample(unique(df$forecaster))
+    colorPalette = setNames(object = viridis(length(unique(df$forecaster))), nm = forecasterRand)
```

Comment:
With viridis, the colors sometimes become quite similar after shuffling.
But it usually gets quite distinct within a few more shuffles!
I personally don't know of other qualitative colorblind-friendly palettes that support the number of categories required.

Reply:
Yeah, I explored the options a bit and thought this one worked the best of the ones I looked at. Definitely open to using another package though if there is a better one.